src/kubeseal_auto/kubeseal.py

```
@@ -138,11 +135,37 @@ def __exit__(self, exc_type: type | None, exc_val: Exception | None, exc_tb: obj
+        """Fall back to system-installed kubeseal binary.
+
+        Raises:
+            BinaryNotFoundError: If kubeseal is not found in PATH.
+
+        """
+        system_binary = shutil.which("kubeseal")
+        if system_binary is None:
+            raise BinaryNotFoundError(
+                "kubeseal binary not found. Please install kubeseal or ensure it's in your PATH. "
+                "See: https://github.com/bitnami-labs/sealed-secrets#installation"
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)